templates/taxi/car_list.html

```diff
@@ -1,7 +1,8 @@
 {% extends "base.html" %}
 
 {% block content %}
-  <h1>Car list</h1>
+  <h1>Car list<a style="float: right" href="car_form.html" class="btn btn-secondary">ADD</a></h1>
+  
   {% if car_list %}
```

Comment:
In Django templates, it's better to use `{% empty %}` clause in `{% for %}` loop instead of checking if the list is empty before the loop. This would make your template code cleaner and easier to read. The `{% empty %}` clause specifies what should be displayed if the iterable is empty.